app/dev-portal/api-catalog-and-packaging.md

```diff
@@ -0,0 +1,183 @@
+* Apply rate limiting policies to an API Package, or per operation
+* Manage role-based access control to specific Developers/Teams
+
+The following table describes common API package use cases:
+<!--vale off-->
+{% table %}
+columns:
+  - title: Use case
+    key: use-case
+  - title: Description
+    key: description
+  - title: Example
+    key: example
+rows:
+  - use-case: Customize APIs for partners
+    description: Using API packages, you can customize which operations/endpoints are included in Dev Portals for partners.
+    example: If you have multiple APIs/services, you can select specific operations from each API, e.g. `GET /billing` and `GET /flights`, creating an API package for partners to an external Dev Portal, titled "Flight Billing API". Meanwhile Billing and Flights APIs can be published to your internal Dev Portal.
+  - use-case: Apply rate limits to Applications
+    description: While Gateway Services or Routes may have systemic rate limits in place, it's often necessary to get lower rate limits to Applications when exposing those capabilities to partners. When createing "Flight Billing API", `GET /billing` may need a lower rate limit of 10 request/minute, while the overall Package could have something higher like 5 requests/second. 
+{% endtable %}
+<!--vale on-->
+
+## How API packages work
+
+API packages are composed from operations. Operations allow you to compose REST-style APIs from a combination of the hostname, path, and method that are processed by the control plane. They are used to map between the operations in your API and Gateway Routes. If an OpenAPI spec is added to the API, the operations are autogenerated. 
+
+The following diagram shows an example of how API packages can be composed from operations:
+
+<!--vale off-->
+{% mermaid %}
+flowchart LR
+ subgraph subGraph0["Control plane"]
+        1["Billing Gateway Service"]
+        2["Flights Gateway Service"]
+  end
+ subgraph subGraph1["Partners API package"]
+        3["POST /flights"]
+        4["GET /flights"]
+  end
+ subgraph subGraph2["Org API package"]
+        5["POST /billing"]
+        6["GET /billing"]
+        7["POST /flights"]
+        8["GET /flights"]
+  end
+    1 --> 5 & 6
+    2 --> 7 & 8 & 3 & 4
+    style subGraph2 fill:#C8E6C9
+    style subGraph1 fill:#BBDEFB
+{% endmermaid %}
+<!--vale on-->
+
+Packaging APIs involves the following steps:
+1. Create an API and attach an OpenAPI spec.
+1. Link a control plane (or control planes) for to allow developer consumption. 
+1. Apply the Access Control Enforcement (ACE) plugin globally.
+1. Create an API package by adding operations and package rate limits. Operations are automatically mapped to Routes using your API's OpenAPI spec or you can create them manually. Gateway configuration is not directly modified, any unmatched operations will be highlighted to indicate action needed by the Gateway operator.
```

Comment:
We will be adding more features for visualizing how operations map to routes in the next iteration. For now I think we should say something more like this:
```suggestion
1. Create an API and attach an OpenAPI spec. Operations from your API's OpenAPI spec should overlap with gateway Routes to ensure requests will be routed to the correct service. Gateway routing configuration is not directly modified by adding operations.
1. Link a control plane (or control planes) for to allow developer consumption. 
1. Apply the Access Control Enforcement (ACE) plugin globally.
1. Create an API package by adding operations and package rate limits.
```

Reply:
I'll add this!